source/GameData.cpp

```diff
@@ -159,16 +161,20 @@ namespace {
 		// Load the icon for the plugin, if any.
 		auto icon = make_shared<ImageSet>(plugin->name);
 
+		string realPath = isZip ? path + "/" + Archive::GetRootPath(path) : path;
+
+		auto Exists = isZip ? &Archive::FileExists : &Files::Exists;
```

Comment:
```suggestion
		bool(const string &) *Exists = isZip ? &Archive::FileExists : &Files::Exists;
```
Is that better? Idk

Reply:
Nah, but it could probably be `const auto`

Reply:
```suggestion
		const auto Exists = isZip ? &Archive::FileExists : &Files::Exists;
```